device-types/Schneider Electric/SpaceLogic-AS-P-NL.yaml

```diff
@@ -0,0 +1,31 @@
+---
+manufacturer: Schneider Electric
+model: SpaceLogic AS-P-NL
+slug: schneider-electric-spacelogic-as-p-nl
+part_number: SXWASPXXX10002
+is_full_depth: false
+u_height: 0
+interfaces:
+  - name: eth1
+    label: Ethernet 1
+    type: 100base-tx
+    enabled: true
+    description: Default DHCP enabled
+  - name: eth2
+    label: Etrhernet 2
+    type: 100base-tx
+    enabled: false
+    description: Default disabled
+console-ports:
+  - name: serial1
+    label: Device
+    type: usb-mini-b
+  - name: serial2
+    label: Host
+    type: usb-micro-a
+power-ports:
+  - name: power
+    label: Power
+    type: other
+    maximum-draw: 10
+comments: '[SpaceLogic™ AS-P-NL Hardware Installation](https://ecostruxure-building-help.se.com/bms/Topics/Show.castle?id=13809)'
```

Comment:
Please move this comment up towards before interfaces

Reply:
@danner26 I would like you to look into incorporating this check as part of the "pre-commit run" book the repository currently includes.

I rely on the `pre-commit run` to be able comply with expectations for PRs before submitting them here on GH. ;-)

Reply:
I agree, not entirely sure how I will work that out but ill see what I can do!